diff --git a/runtime/bin/platform_win.cc b/runtime/bin/platform_win.cc
index 5176c5e..65f2b12 100644
--- a/runtime/bin/platform_win.cc
+++ b/runtime/bin/platform_win.cc
@@ -218,14 +218,7 @@
   if (!SocketBase::Initialize()) {
     return false;
   }
-  // 256 is max length per https://learn.microsoft.com/en-us/windows/win32/api/winsock2/nf-winsock2-gethostnamew#remarks
-  const int HOSTNAME_MAXLENGTH = 256;
-  wchar_t hostname_w[HOSTNAME_MAXLENGTH];
-  if (GetHostNameW(hostname_w, HOSTNAME_MAXLENGTH) != 0) {
-    return false;
-  }
-  return WideCharToMultiByte(CP_UTF8, 0, hostname_w, -1, buffer, buffer_length,
-                             nullptr, nullptr) != 0;
+  return gethostname(buffer, buffer_length) == 0;
 #endif
 }
 
